### PR TITLE
Add changelog fragment for #26

### PR DESCRIPTION
**Pull request by Jorge-Rodriguez**
##### SUMMARY
Add missing changelog for changes introduced in #26

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
mysql_user

### CODE REVIEW

**Comment by Jorge-Rodriguez** [comment]
Ping @Andersson007

File: changelogs/fragments/26-remove_tls_requirements.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+bugfixes:
+- mysql_user - Correct procedure to check existing TLS requirements.
+- mysql_user - Minor syntax fixes.
+improvements:
+- mysql_user - Added tests to verify that TLS requirements are removed with an empty `tls_requires` parameter.
```
Comment (Andersson007) [inline changelogs/fragments/26-remove_tls_requirements.yml]:
```suggestion
- mysql_user - added tests to verify that TLS requirements are removed with an empty ``tls_requires`` option (https:// link to the reported issue).
```

@Jorge-Rodriguez thanks! please, change the other entry as well, https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to

File: changelogs/fragments/26-remove_tls_requirements.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- mysql_user - Added examples on how to remove TLS requirements by setting an empty `tls_requires` parameter.
+- mysql_user - Refactored TLS requirement tests into their own file.
```
Comment (Andersson007) [inline changelogs/fragments/26-remove_tls_requirements.yml]:
```suggestion
```
we don't need fragments for doc changes and tests, it's necessary only for things that can affect module/plugin behavior

File: changelogs/fragments/26-remove_tls_requirements.yml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/26-remove_tls_requirements.yml]: we should use `option` instead of parameter / argument according to the doc best practices.

Comment (Jorge-Rodriguez) [reply changelogs/fragments/26-remove_tls_requirements.yml]: I think I've used parameter in a number of places, let me review that.

**Comment by Jorge-Rodriguez** [comment]
@Andersson007 does removing the improvements section address all your comments on this PR, or did I miss something?

File: changelogs/fragments/26-remove_tls_requirements.yml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/26-remove_tls_requirements.yml]: it's just a best practice in Ansible. i think we should follow it as the others practices.

**Comment by Andersson007** [comment]
> @Andersson007 does removing the improvements section address all your comments on this PR, or did I miss something?

not sure, i understand correctly:) my first comments about formatting of `bugfixes` are still relevant:)

File: changelogs/fragments/26-remove_tls_requirements.yml (hunk 1 above)
Comment (Andersson007) [reply changelogs/fragments/26-remove_tls_requirements.yml]: If it's interesting to you https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#style-guide and also the several references at the page's bottom

**Comment by Jorge-Rodriguez** [comment]
I must be really slow today. Didn't your formatting comments have to do with replacing parameter with option? Or do you mean the capitalization?

**Comment by Andersson007** [comment]
> I must be really slow today. Didn't your formatting comments have to do with replacing parameter with option? Or do you mean the capitalization?

The entries should look like:
`- mysql_user - added tests to verify that TLS requirements are removed with an empty ``tls_requires`` option (https:// link to the reported issue).`
1) the description should start with a lowercase latter
2) the parameter name should be between `` on both sides
3) put the issue link between the parentheses / if there's no issue - a reference to the PR itself :)

**Comment by Jorge-Rodriguez** [comment]
Thank you. I feel like I woke up with only half a brain. I'll get to that.

**Comment by Andersson007** [comment]
> Thank you. I feel like I woke up with only half a brain. I'll get to that.

Np, don't worry, this is my usual condition;), so fully understood:) Feel free to ask whenever any clarifications needed!

**Comment by Jorge-Rodriguez** [comment]
@Andersson007 I hope this is good now. I added the line about the tests because that addresses issue #20 directly

File: changelogs/fragments/26-remove_tls_requirements.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+bugfixes:
+- mysql_user - added tests to verify that TLS requirements are removed with an empty ``tls_requires`` option (https://github.com/ansible-collections/community.mysql/issues/20).
+- mysql_user - correct procedure to check existing TLS requirements (https://github.com/ansible-collections/community.mysql/pull/26).
+- mysql_user - minor syntax fixes.
```
Comment (Andersson007) [inline changelogs/fragments/26-remove_tls_requirements.yml]:
```suggestion
- mysql_user - minor syntax fixes (https://github.com/ansible-collections/community.mysql/pull/26).
```

**Comment by Andersson007** [comment]
Thank you!

**Comment by Andersson007** [comment]
Fyi, I’m going to release the collection until Wednesday. So if you have anything urgent, we have time until that

**Comment by Jorge-Rodriguez** [comment]
> Fyi, I’m going to release the collection until Wednesday. So if you have anything urgent, we have time until that

I can have a stab at #28 but I can't guarantee that it'll be ready on time for the release

**Comment by Andersson007** [comment]
I think it’s not a blocker, take your time

**Comment by Andersson007** [comment]
With collections we can release every day if we need;)

**Pull request merged**